### PR TITLE
WIP Introduce OpenAPI helper to add more response statuses

### PR DESCRIPTION
**Pull request by sultaniman**
This PR introduces enhancements to OpenAPI generation, adds additional responses to swagger schema  https://fastapi.tiangolo.com/advanced/additional-responses/?h=responses

### CODE REVIEW

File: server/polar/kit/api/responses.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict, Union
+
+from starlette.exceptions import HTTPException
+
+
+def get_api_errors(*additional_responses) -> Dict[Union[int, str], Dict[str, Any]]:
```
Comment (Kludex) [inline server/polar/kit/api/responses.py:6]: This is not needed... 👀

Comment (sultaniman) [reply server/polar/kit/api/responses.py:6]: @Kludex thanks for taking a look 🤝 , how can we change this what do you suggest?

Comment (Kludex) [reply server/polar/kit/api/responses.py:6]: Just create the dictionary on each endpoint

Comment (birkjernstrom) [reply server/polar/kit/api/responses.py:6]: Thanks @Kludex for the great feedback ♥️ I agree with it that we can simply add the responses directly to the router decorator, i.e no need for the `get_api_errors` function. @sultaniman Thanks for helping out with this and improving the documentation/Swagger of our endpoints – much appreciated 🙏🏼

Comment (sultaniman) [reply server/polar/kit/api/responses.py:6]: @Kludex @birkjernstrom then I will close this PR and will instead open a separate one.

Comment (birkjernstrom) [reply server/polar/kit/api/responses.py:6]: @sultaniman Awesome 🙏🏼 You can convert it to a draft too and ping me at any point if you want an earlier review too – whatever you prefer 🙂

Comment (birkjernstrom) [reply server/polar/kit/api/responses.py:6]: Oops. Saw now that you closed it. I was too slow :)

**Pull request closed without merging**